pod/perldelta.pod

```diff
@@ -59,6 +59,13 @@ XXX For a release on a stable branch, this section aspires to be:
 
 [ List each incompatible change as a =head2 entry ]
 
+=head2 sort with no arguments is now a compile-time error
+
+C<@a = sort;> used to be equivalent to C<@a = ();>, which wasn't very
```

Comment:
Technically, the C<> should be surrounded by S<>, but it doesn't matter here since it is first in the paragraph, so won't wrap.